apollo-runtime/src/main/java/com/apollographql/android/ApolloCall.java

```diff
@@ -10,12 +10,18 @@
 
 public interface ApolloCall {
 
-  void cancel();
-
   @Nonnull <T extends Operation.Data> Response<T> execute() throws IOException;
 
   @Nonnull <T extends Operation.Data> ApolloCall enqueue(@Nullable Callback<T> callback);
 
+  @Nonnull ApolloCall network();
```

Comment:
Not sure if we need this now, as all these subclasses are different from each other from cachePolicy flag only. I guess we can review this later if we really need to introduce them